src/rgw/const.py

```diff
@@ -21,7 +21,7 @@
 SERVICE_NAME = f'{COMPONENT_NAME}_setup' # rgw_setup
 INSTALL_PATH = '/opt/seagate/cortx'
 RGW_INSTALL_PATH = f'{INSTALL_PATH}/{COMPONENT_NAME}'
-ADMIN_CREATION_TIMEOUT = 60
+ADMIN_CREATION_TIMEOUT = 600
```

Comment:
Add 
#TODO Revisit after Motr delay issue resolved, seen while admin creation, for CORTX cluster with 15 nodes

Reply:
Done.